ckanext/xloader/jobs.py

```diff
@@ -170,12 +170,16 @@ def _get_static_validation_options():
         return static_validation_options
 
     def direct_load():
+        static_options = _get_static_validation_options()
         fields = loader.load_csv(
             tmp_file.name,
             resource_id=resource['id'],
             mimetype=resource.get('format'),
             # (canada fork only): adds in dialect argument to pass static dialect
-            dialect=_get_static_validation_options().get('dialect', {})
+            dialect=static_options.get('dialect', {})
```

Comment:
Doesn't this need a backslash to continue the line?

Reply:
it's in the parenthesis of `load_csv`, so it is fine